Gemfile

```diff
@@ -8,7 +8,8 @@ gem 'rails', '~> 6.1.4'
 # Variables can be overridden for local dev in Gemfile-dev
 @hostdata_gem ||= { github: '18F/identity-hostdata', tag: 'v3.4.0' }
 @logging_gem ||= { github: '18F/identity-logging', tag: 'v0.1.0' }
-@saml_gem ||= { github: '18F/saml_idp', tag: '0.15.0-18f' }
+# @saml_gem ||= { github: '18F/saml_idp', tag: '0.15.0-18f' }
```

Comment:
TODO: replace once https://github.com/18F/saml_idp/pull/51 is merged and the tag is pushed